src/components/Cluster/ClusterDetail/RegionAndVersions.js

```diff
@@ -85,12 +87,9 @@ function RegionAndVersions({
       </div>
       {canUpgrade && (
         <a className='upgrade-available' onClick={showUpgradeModal}>
-          <Upgrade>
-            <span>
-              <i className='fa fa-warning' />
-              Upgrade available
-            </span>
-          </Upgrade>
+          <UpgradeWrapperDiv>
```

Comment:
Any reason we can't use the new component here? Instead making a new styled component and injecting the styles?